CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [NEXT]
+- feat(fabric.Text): Adding support for RTL languages by adding `direction` property [#7046](https://github.com/fabricjs/fabric.js/pull/7046)
```

Comment:
we miss all the list of stuff from 4.4.0 t0 today.
And next becomes 4.5.0

![image](https://user-images.githubusercontent.com/1194048/119158875-f8db2f80-ba56-11eb-8a42-5faaa7cb5f8e.png)

not much.
I m also sure this can be automated with a script at some point on the road of become a normal library

Reply:
Got you.  Will fix ASAP.

Reply:
fixed